attunement.lic

```diff
@@ -22,10 +23,13 @@ class Attunement
       pause 15
```

Comment:
Should MMs get special treatment here, or adhere to the room list below and use the guild to determine `perc` vs `perc mana`?